service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -650,11 +647,8 @@ public void setTariffLimits(Long tariffId, SetTariffLimitsDto setTariffLimitsDto
                 throw new BadRequestException(ErrorMessage.MAX_PRICE_VALUE_IS_INCORRECT);
             }
 
-            tariffsInfo.setMinAmountOfBigBags(null);
-            tariffsInfo.setMaxAmountOfBigBags(null);
-
-            tariffsInfo.setMinPriceOfOrder(setTariffLimitsDto.getMinPriceOfOrder());
-            tariffsInfo.setMaxPriceOfOrder(setTariffLimitsDto.getMaxPriceOfOrder());
+            tariffsInfo.setMinQuantity(setTariffLimitsDto.getMinPriceOfOrder());
+            tariffsInfo.setMaxQuantity(setTariffLimitsDto.getMaxPriceOfOrder());
```

Comment:
please discuss it with Viktoriya
I think we should not set `priceOfOrder` to `quantity`